_posts/2020-09-16-Buildah-version-v1.16.0.md

```diff
@@ -0,0 +1,121 @@
+  * The handling of seccomp is now done in the containers/common project.
+  * The  `--timestamp` option has  been added to the `bud` and `commit` commands to allow the ‘create’ timestamp to be set to seconds since epoch, replacing the `--omit-timestamp` option.  See the respective man pages for more information.
+  * The `--quiet` option should be more quiet.
+  * Fix ownership of content copied using `COPY --from`, prior ownership information on files was being dropped.
+  * Error handling was improved in the `run` command and a few messages were clarified.
+  * A copier package is now used internally to rework the handling of caches during `ADD` and `COPY` operations.
+  * When doing a `COPY` of an archive file, the destination file name was incorrectly being blanked out; this has been corrected.
+  * The retry functions have been replaced by functions from the common/pkg/retry package.
+  * A number of timestamp comparisons were being done with `==` and they have been converted to use `time.Time.Equal()` which is more accurate.
+  * Fixed errors found in a Coverity scan.
+  * Namespace handling options in the `bud`, `from` and `run` commands have been changed to match the same options in Podman.  In addition the `--network` option for `bud` now accepts the same values that the corresponding Podman command does.
+  * A dependency on the xz package has been added to Buildah images that are built for [quay.io](https://quay.io/buildah).
+  * Storage was not always shutdown as it should have been on error, which could lead to a leaked mount point.  This has been corrected.
+  * The `COPY --from` command now works when an argument is given to it.
+  * The version of Buildah used to build an image is now embedded in the  BuilderIdentityAnnotation within the image.
+  * The /etc/host and /etc/resolv.conf files are no longer bound if network is not present in the container.
+  * An unnecessary call to the function NewImage() has been removed from the build processing.
+  * When processing multiple archives during a .dockerignore process, the processing would stop at the end of the first archive.  Now all archives are processed as they should be.
+  * Fixed & added notes regarding problematic language in the codebase that were not inclusive.
+  * Added a dependency on github.com/stretchr/testify/require.
+  * The build processing now waits for stages that might not have even started yet instead of trying to continue and then failing.
+  * Mounts under `/sys` were not always accessible to rootless users.  This has been corrected.
+  * Environment variables can now be pre-declared in the containers.conf(5) file.
+  * The right stage's image is now returned as the "final" image.
+  * When build arguments and environment variables have duplicate names, the values are now deduplicated.  Build arguments override Default arguments and Environment variables set during the build process overrides both of those.
```

Comment:
@nalind can you triple check that I have this ordering correct here, please?

Reply:
Yes, that's the correct ordering.  That last "overrides" should be "override" (subject-verb agreement).

Reply:
TY and tweaked.